src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/SpecifyIFormatProvider.cs

```diff
@@ -86,12 +86,19 @@ protected override void InitializeWorker(CompilationStartAnalysisContext context
             var charType = context.Compilation.GetSpecialType(SpecialType.System_Char);
             var boolType = context.Compilation.GetSpecialType(SpecialType.System_Boolean);
             var guidType = context.Compilation.GetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemGuid);
+            var nullableTType = context.Compilation.GetSpecialType(SpecialType.System_Nullable_T);
+            INamedTypeSymbol? GetNullableType(INamedTypeSymbol? typeSymbol) => typeSymbol is not null ? nullableTType?.Construct(typeSymbol) : null;
```

Comment:
Kindly move the local function towards the end of the method.

Reply:
I ended up removing the local function.